css/receiving-rides.css

```diff
@@ -26,7 +32,189 @@
     color: white;
     font-family: 'Open Sans', sans-serif;
     border-radius: 36px;
+    font-size: 1.8rem;
+    border: none;
+    padding: 1rem 1.5rem;
+    margin: 1rem;
+}
+
+.steps-section {
+    display: flex;
+    justify-content: center;
+    font-family: 'Open Sans', sans-serif;
+}
+
+.steps-content {
+    width: 70%;
+}
+
+.steps-header {
+    padding: 3rem 0rem;
+}
+
+.steps-body {
+    display: flex;
+    justify-content: space-between;
+}
+
+.steps-timeline {
+     width: 70%;
+     height: auto;
+}
+
+.steps-name {
+    font-size: 2.5rem;
+    font-style: normal;
+    font-weight: normal;
+    color: #5C6E93;
+    padding-bottom: 1rem;
+}
+
+.steps-underline {
+    border-bottom: 1px solid black;
+    width: 10rem;
+}
+
+.steps-img {
+    width: 25%;
+    height: 70%;
+}
+
+.steps-timeline {
+    padding-left: 2rem;
+    margin-bottom: 5rem;
+}
+
+.steps-bullet {
+    position: relative;
+    margin: 0;
+    padding-bottom: 1rem;
+    list-style-type: square;
+    font-size: 4rem;
+}
+
+.steps-bullet:before {
+    width: 1px;
+    height: 100%;
+    content: "";
+    position: absolute;
+    left: -3.1rem;
+    border-left: 1px solid black;
+}
+
+.steps-bullet:first-child:before {
+    top: 3rem;
+}
+
+.steps-bullet:last-child:before {
+    height: 3.5rem;
+}
+
+.steps-subtitle {
+    font-style: normal;
+    font-weight: normal;
```

Comment:
delete (default is normal so don't need to specify - delete other places this is done as well)